crates/wasi-http/src/types.rs

```diff
@@ -167,6 +177,22 @@ pub fn default_send_request(
     Ok(fut)
 }
 
+pub fn default_is_forbidden_request_header(name: &HeaderName) -> bool {
+    use hyper::header;
+
+    const FORBIDDEN: &'static [HeaderName] = &[header::TRANSFER_ENCODING];
+
+    FORBIDDEN.contains(name)
+}
+
+pub fn default_is_forbidden_response_header(name: &HeaderName) -> bool {
+    use hyper::header;
+
+    const FORBIDDEN: &'static [HeaderName] = &[header::TRANSFER_ENCODING];
+
+    FORBIDDEN.contains(name)
+}
+
```

Comment:
I'd appreciate some input on what a good default set of forbidden headers is for requests and responses.